lakefile.toml

```diff
@@ -21,3 +21,6 @@ git = "https://github.com/PatrickMassot/checkdecls.git"
 [[lean_lib]]
 name = "FLT"
 globs = ["FLT", "FermatsLastTheorem"]
+
+[[dependancies.FLT]]
```

Comment:
Are you sure this works, given the typo-ed "dependencies"? (I don't know if this is an actual section of `lakefile`s.)